lib/matplotlib/tests/test_axes.py

```diff
@@ -2108,68 +2108,59 @@ def layers(n, m):
     axs[1, 1].stackplot(range(100), d.T, baseline='weighted_wiggle')
 
 
+def _bxp_test_helper(
+        stats_kwargs={}, transform_stats=lambda s: s, bxp_kwargs={}):
+    np.random.seed(937)
+    logstats = matplotlib.cbook.boxplot_stats(
+        np.random.lognormal(mean=1.25, sigma=1., size=(37, 4)), **stats_kwargs)
+    fig, ax = plt.subplots()
+    if bxp_kwargs.get('vert', True):
+        ax.set_yscale('log')
+    else:
+        ax.set_xscale('log')
+    # Work around baseline images generate back when bxp did not respect the
+    # boxplot.boxprops.linewidth rcParam when patch_artist is False.
+    if not bxp_kwargs.get('patch_artist', False):
```

Comment:
OK, but you never test the new proper behaviour this way, do you?

Reply:
indeed.  slightly changed the test setup to test that the rc is used.